internal/build/docker_test.go

```diff
@@ -236,6 +237,19 @@ func TestEntrypoint(t *testing.T) {
 	f.assertFilesInImageWithContents(string(d), contents)
 }
 
+func TestDockerfileWithEntrypointNotPermitted(t *testing.T) {
+	f := newTestFixture(t)
+	defer f.teardown()
+
+	df := `FROM alpine
+ENTRYPOINT ["sleep", "100000"]`
+
+	_, err := f.b.BuildDocker(context.Background(), df, []tiltd.Mount{}, []tiltd.Cmd{}, tiltd.Cmd{})
```

Comment:
This could have failed for any number of reasons, should we assert a bit on the error type/message? Or test `checkDockerfileForEntrypoint` directly?

Reply:
better?